coderd/workspaceagentsrpc.go

```diff
@@ -143,6 +143,7 @@ func (api *API) workspaceAgentRPC(rw http.ResponseWriter, r *http.Request) {
 		DerpForceWebSockets:       api.DeploymentValues.DERP.Config.ForceWebSockets.Value(),
 		DerpMapUpdateFrequency:    api.Options.DERPMapUpdateFrequency,
 		ExternalAuthConfigs:       api.ExternalAuthConfigs,
+		Experiments:               api.Experiments,
```

Comment:
Is this still needed?

Reply:
Yes we pass it into the AgentAPI -> StatsAPI which uses it to zero out session data from incoming stats when the experiment is enabled.  https://github.com/coder/coder/pull/13637/files/61648cbca9078d66e86d9c99f3385f7b6ec1ad55#diff-4d8459846359fa6e667e4d51d95f3eb28aa37a75039d7d08084accaaefeeb6fbR60